src/cluster-client/src/client.proto

```diff
@@ -19,10 +19,10 @@ message ProtoClusterStartupEpoch {
 }
 
 message ProtoTimelyConfig {
+    reserved 5;
     uint64 workers = 1;
     uint64 process = 2;
     repeated string addresses = 3;
     uint32 idle_arrangement_merge_effort = 4;
-    bool variable_length_row_encoding = 5;
     uint32 arrangement_exert_proportionality = 6;
```

Comment:
We don't require backwards-compatibility for this message, so you can just re-use 5.